docs/apis-tools/grpc.md

```diff
@@ -150,6 +160,7 @@ message CancelProcessInstanceResponse {
 Returned if:
 
 - No process instance exists with the given key. Note that since process instances are removed once they are finished, it could mean the instance did exist at some point.
+- No process instance exists with the given key for the tenants the user is authorized to work with.
```

Comment:
💭 Just thinking out loud. I wonder if we even need to specify this case. User makes a request, we can't find anything for the given tenant, so we just say it doesn't exist. If it's because the tenant doesn't match or not is up to the user.

Either way, it can't hurt having it here.

Reply:
@remcowesterhoud my reasoning for adding this line was to avoid confusion/support cases with users who may have had their authorization to a tenant revoked, or maybe decided to use a different token that is missing the appropriate tenant authorization.

In the scenarios above, this line will (hopefully) trigger a check for the authorizations, instead of users thinking it's a bug because a PI that they know was started can't be found.

WDYT?

Reply:
Yeah I think it's fair. Like I said it can't hurt having it here and it's already written so no point in taking it out. Worst that can happen is that it helps someone 😄